src/Miunie.Core/Services/ProfileService.cs

```diff
@@ -25,7 +28,11 @@ public async Task GiveReputation(MiunieUser invoker, MiunieUser target, MiunieCh
                 return;
             }
 
-            if (_reputationProvider.AddReputationHasTimeout(invoker, target)) { return; }
+            if (_reputationProvider.AddReputationHasTimeout(invoker, target))
+            {
+                _logger.Log($"User '{invoker.Name}' has a reputation timeout for User '{target.Name}', ignoring...");
```

Comment:
This doesn't execute in my machine
![image](https://user-images.githubusercontent.com/13708100/55279062-73dfaa00-5314-11e9-8af4-31180f7e2b9e.png)
The bot do nothing whenever i try to add rep to an user twice. With nothing i mean no output in discord, nor in the console